content/en/opentelemetry/agent/install_agent_with_collector.md

```diff
@@ -0,0 +1,577 @@
+  batch:
+    timeout: 10s
+connectors:
+  datadog/connector:
+    traces:
+      compute_top_level_by_span_kind: true
+      peer_tags_aggregation: true
+      compute_stats_by_span_kind: true
+service:
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [infraattributes, batch]
+      exporters: [datadog, datadog/connector]
+    metrics:
+      receivers: [otlp, datadog/connector, prometheus]
+      processors: [infraattributes, batch]
+      exporters: [datadog]
+    logs:
+      receivers: [otlp]
+      processors: [infraattributes, batch]
+      exporters: [datadog]
+
+{{< /code-block >}}
+
+#### Key components
+
+To send telemetry data to Datadog, the following components are defined in the configuration:
+
+{{% collapse-content title="Component diagram" level="p" %}}
+{{< img src="/opentelemetry/embedded_collector/components.png" alt="Diagram depicting the Agent deployment pattern" style="width:100%;" >}}
+{{% /collapse-content %}}
+
+##### Datadog connector
+
+The [Datadog connector][6] computes Datadog APM Trace Metrics.
+
+{{< code-block lang="yaml" filename="otel-config.yaml" disable_copy="false" collapsible="true" >}}
+connectors:
+  datadog/connector:
+    traces:
+      compute_top_level_by_span_kind: true
+      peer_tags_aggregation: true
+      compute_stats_by_span_kind: true
+{{< /code-block >}}
+
+##### Datadog exporter
+
+The [Datadog exporter][7] exports traces, metrics, and logs to Datadog.
+
+{{< code-block lang="yaml" filename="otel-config.yaml" disable_copy="false" collapsible="true" >}}
+exporters:
+  datadog:
+    api:
+      key: ${env:DD_API_KEY}
+{{< /code-block >}}
+
+##### Prometheus receiver
+
+The [Prometheus receiver][8] collects health metrics from the OpenTelemetry Collector for the Metrics pipeline.
+
+{{< code-block lang="yaml" filename="otel-config.yaml" disable_copy="false" collapsible="true" >}}
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: "otelcol"
+          scrape_interval: 10s
+          static_configs:
+            - targets: ["0.0.0.0:8888"]
+{{< /code-block >}}
+
+For more information, see the [Collector Health Metrics][8] documentation.
+
+### Deploy the Agent with the OpenTelemetry Collector
+
+To install or upgrade the Datadog Agent with OpenTelemetry Collector in your Kubernetes environment, use the following Helm command:
+
+```shell
+helm upgrade -i <RELEASE_NAME> datadog/datadog -f datadog-values.yaml
+```
+
+This Helm chart deploys the Datadog Agent with OpenTelemetry Collector as a DaemonSet. The Collector is deployed on the same host as your application, following the Agent deployment pattern. The Gateway deployment pattern is not supported.
+
+{{% collapse-content title="Deployment diagram" level="p" %}}
+{{< img src="/opentelemetry/embedded_collector/deployment.png" alt="Diagram depicting the Agent deployment pattern" style="width:100%;" >}}
+{{% /collapse-content %}}
+
+## Instrument and configure the application
```

Comment:
Is this the correct title for this section? Should it be more like "Send your telemetry data to Datadog"?